app/forms/idv/profile_form.rb

```diff
@@ -11,6 +11,15 @@ def self.model_name
 
     validate :ssn_is_unique, :dob_is_sane
 
+    validates_format_of :zipcode,
+                        with: /\A\d{5}(-\d{4})?\z/,
+                        message: I18n.t('idv.errors.bad_zipcode'),
+                        allow_blank: true
+    validates_format_of :ssn,
+                        with: /\A\d{3}-\d{2}-\d{4}\z/,
```

Comment:
Rethinking this regex, I think we should update it to make the hyphens optional here and on the frontend, just because in the scenario where js isn't enabled and those hyphens aren't automatically inserted, I don't think we _have_ to have them, unless the third party proofer is expecting them in a particular format?  @pkarman do you know if a social and/or zip can just be a string of numbers, or do they need to include any additional formatting?